src/CaptureClient/CaptureClient.cpp

```diff
@@ -84,6 +85,7 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> CaptureClient::Capture(
   }
 
   state_ = State::kStarting;
+  LOG("State is now kStarting");
```

Comment:
Was this added as debug output or do you want to keep it? (Seems like it could be useful to have in the log.)

Reply:
Yep, it could be useful.